README.md

```diff
@@ -24,15 +24,8 @@ The documentation is generated from the following resources
 - [CDK components, services, and directives](https://github.com/angular/material2/tree/master/src/cdk)
 
 ## Development Setup
-1. Clone Angular Material in the parent directory of this repo
-    1. `cd ..`
-    1. `git clone git@github.com:angular/material2.git`
 1. Install Gulp globally: `npm i -g gulp`
```

Comment:
Shouldn't this line be removed as well? Do we still need gulp?

Reply:
Gulp is no longer used so it should be removed too.